openwisp_radius/settings.py

```diff
@@ -40,6 +40,11 @@ def get_settings_value(option, default):
 BATCH_MAIL_SENDER = get_settings_value('BATCH_MAIL_SENDER', settings.DEFAULT_FROM_EMAIL)
 API_AUTHORIZE_REJECT = get_settings_value('API_AUTHORIZE_REJECT', False)
 SOCIAL_LOGIN_ENABLED = 'allauth.socialaccount' in settings.INSTALLED_APPS
+SAML_LOGIN_ENABLED = 'djangosaml2' in settings.INSTALLED_APPS
+SAML_REGISTRATION_METHOD_LABEL = get_settings_value(
+    'SAML_REGISTRATION_METHOD_LABEL', _('Single Sign-On (SAML)')
+)
+SAML_IS_VERIFIED = get_settings_value('SAML_IS_VERIFIED', False)
```

Comment:
can you please implement (and document) this to be overridable from the `OrganizationRadiusSettings` please (not urgent)?

Reply:
Ignore this comment, since saml is optional, probably this is overkill now.